### PR TITLE
Fixes changelog style #1395 🚀

### LINKED ISSUE #1395: Update Changelog.md to our newest standards.
We've updated our guidelines for a Changelog, [here](https://github.com/Moya/contributors/blob/master/Changelog%20Guidelines.md). As you can see in our [Changelog](https://github.com/Moya/Moya/blob/master/Changelog.md), we started using the newest guidelines as of Moya version 10. We need to update the older entries of the Changelog to the newest standards as well.

If you want to claim this issue, please comment here or assign yourself if you are part of Moya organization already.

If there are any questions, we are happy to assist!

### PR DESCRIPTION
**Pull request by kimdv**
Hi! 

Startet formatting changelog, hope you can use it 🎉

Resolves #1395

### CODE REVIEW

**Comment by SD10** [comment]
This is awesome @kimdv 💯 Thank you so much for taking the time to do this 😃 👍

**Comment by AndrewSB** [comment]
Closes #1395

**Comment by sunshinejr** [comment]
@AndrewSB I’m afraid we don’t have one at the moment. Probably need to add them at some point, though.

File: Changelog.md [hunk 1]
```diff
@@ -199,83 +234,95 @@ structure and more.
...
 
-# 5.2.0
+### Fixed
+- Fixes cases where underlying network errors were not properly propagated.
 
-- Updated to RxSwift 2.0.0-beta.4
+# [5.2.0] - 2015-12-xx
 
-# 5.1.0
+- Updated to RxSwift 2.0.0-beta.4
```
Comment (SD10) [inline Changelog.md]: Can we add a `Changed` header here?

File: Changelog.md [hunk 2]
```diff
@@ -199,83 +234,95 @@ structure and more.
...
-# 6.1.1
-
+# [6.1.1] - 2016-01-17
+### Added
 - Compatibility with RxSwift 2.1.x.
 
-# 6.1.0
-
+# [6.1.0] - 2016-01-26
+###
```
Comment (SD10) [inline Changelog.md]: Can we add a `Changed` header here?

File: Changelog.md [hunk 3]
```diff
@@ -199,83 +234,95 @@ structure and more.
...
-# 6.5.0
-
+# [6.5.0] - 2016-05-26
+### Added
 - Added `queue` parameter to `request` and `sendRequest`. This open up option to use other queue instead of main queue for response callback.
 
-# 6.4.0
+# [6.4.0] - 2016-04-02
 
 - Makes `convertResponseToResult` public to make use of this method when dealing with Alamofire directly
```
Comment (SD10) [inline Changelog.md]: Can we add these 3 things under `Changed`?

File: Changelog.md [hunk 4]
```diff
@@ -63,57 +68,69 @@
...
+### Added
 - Supported the Swift package manager
 - Added `AccessTokenPlugin` for easier authorization.
 - Added `AccessTokenAuthorizable` protocol for optionally controlling the authorization behavior of `TargetType`s when using `AccessTokenPlugin`.
 - Added availability tags for renamed functions included in the Swift 3 migration.
 
-# 8.0.0-beta.6
+###
+- `Moya.Error` now conforms to `LocalizedError` protocol.
+- Added documentation for `TargetType` and associated data structures.
```
Comment (SD10) [inline Changelog.md]: Can we move this up to added?

File: Changelog.md [hunk 5]
```diff
@@ -63,57 +68,69 @@
...
+- Re-add `MultiTarget` to project.
```
Comment (SD10) [inline Changelog.md]: Can we move this up to added?

File: Changelog.md [hunk 6]
```diff
@@ -63,57 +68,69 @@
...
-- Moved tests to Moya.xcodeproj.
+
+### Added
 - Supported the Swift package manager
 - Added `AccessTokenPlugin` for easier authorization.
 - Added `AccessTokenAuthorizable` protocol for optionally controlling the authorization behavior of `TargetType`s when using `AccessTokenPlugin`.
 - Added availability tags for renamed functions included in the Swift 3 migration.
 
-# 8.0.0-beta.6
+###
```
Comment (SD10) [inline Changelog.md]: Lets put `Changed` here after moving the two items below

File: Changelog.md [hunk 7]
```diff
@@ -63,57 +68,69 @@
...
+- Updated minimum version of `RxSwift` to `3.3`.
+- Updated minimum version of `ReactiveSwift` to 2.0.
 
+# [8.0.5] - 2017-05-26
+### Fixed
 - Fixed a bug where you would have two response events in `requestWithProgress` method on RxMoya module.
 
-# 8.0.4
+# [8.0.4] - 2017-05-09
+###Changed
```
Comment (SD10) [inline Changelog.md]: Remove the space to fix the header

**Comment by kimdv** [comment]
@SD10 the changes should be added now 🚀

**Comment by ashfurrow** [comment]
Thanks a lot for contributing to Moya! I've invited you to join the Moya GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/Moya/contributors) and feel free to reach out with any questions.

**Pull request merged**